bittensor/core/async_subtensor.py

```diff
@@ -1611,6 +1613,86 @@ async def set_weights(
 
         return success, message
 
+    async def batch_set_weights(
+        self,
+        wallet: "Wallet",
+        netuids: list[int],
+        uidss: list[Union[NDArray[np.int64], "torch.LongTensor", list]],
+        weightss: list[Union[NDArray[np.float32], "torch.FloatTensor", list]],
```

Comment:
Typos? Also is there a reason we accept `torch.LongTensor` objects for `set_weights` but not for `commit_weights`?

Reply:
> Typos?
Supposed to be a list of uid lists. Maybe `uids_list`?

Reply:
> Also is there a reason we accept `torch.LongTensor` objects for `set_weights` but not for `commit_weights`?

I have no idea. This is copy-pasted from the previous impl